images/clusterloader2/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM golang:1.14.4 AS clusterloader-builder
```

Comment:
If you name this "builder" instead of "clusterloader-builder", it's much more copy-pastable for someone in the future.

Reply:
+1